gradle.properties

```diff
@@ -116,7 +116,7 @@ artifactRedirecting.androidx.annotation.version=1.9.1
 artifactRedirecting.androidx.graphics.version=1.0.1
```

Comment:
The last merged to `integration` commit is behind `Savedstate 1.3.0-alpha06` commit

<img width="1040" alt="image" src="https://github.com/user-attachments/assets/b39d1836-3909-4e19-a406-db96b3bcbc4e" />

Before merging this PR, we have to merge `Savedstate 1.3.0-alpha06` commit into `integration` to avoid future merge conflicts.

Reply:
Or include `lifecycle` as ignored folder in https://github.com/JetBrains/compose-multiplatform-core/blob/85ce72d2586f7a2c8885deb3dbabb6c4799e920d/scripts/mergeAOSP.sh#L34 as we done with `navigation`

Reply:
wait a moment :) the PR is to the integration branch

Reply:
Agree with Igor, I missed that.

> Or include lifecycle as ignored folder in

only after sync to up-to-date state, so not in this PR for sure

Reply:
Because we excluded this folder from the main merge (`mergeAOSP.sh`), we can merge it separately from any commit now.